sphinx_immaterial/inline_icons.py

```diff
@@ -0,0 +1,57 @@
+"""This module adds the md-icon role. This is specific to this this theme because,
```

Comment:
Perhaps we should use `im-icon` or `im:icon` rather than `md` (for mkdocs)?

Reply:
I was thinking of maybe `si` for sphinx-immaterial. I only went with `md` because the other custom directives start with that (which I had assumed meant material design).

Reply:
si sounds good to me

Reply:
I'll also change the `.md-icon-inline` CSS class to `.si-icon-inline`. The `.md-icon` CSS class is something we get from upstream.